tests/vectors/resolution/afterUpdate.json

```diff
@@ -18,7 +18,6 @@
     "verificationMethod": [
       {
         "id": "#publicKeyModel1Id",
-        "controller": "",
```

Comment:
please read did core before making changes to rest vectors.... https://w3c.github.io/did-core/#verification-methods

> The properties MUST include the id, type, controller, and specific verification method properties, and MAY include additional properties.

I ideally you would also have tests that showed that JSON-LD was valid.

Reply:
This change is my fault. I read the spec but I looked at wrong section (controller at top level of the document). Sorry about this. Can the controller be empty though?

Reply:
the controller should equal `didDocument.id` in order for verifiable credentials to be valid.... see: https://github.com/OR13/did-params-and-you/blob/master/examples/case-0-legal-working.json#L35

and our version of sidetree: https://github.com/transmute-industries/sidetree.js/blob/main/packages/core/src/DocumentComposer.ts#L65

If his repo had CI tests that used the result of test vectors, with verifiable credentials, it would be obvious when breaking changes like this were introduced.

btw, the reason that "" is legal here, is because of `@base` / relative IRIs in sidetree did documents....

You can check the nquads to see that its the same...

if the iri is blank, it will get mapped to the didDocument.id... making what we had before legal (but only for relative ref based did documents).

Reply:
So it is legal because it will be populated when processing @base - thank you for this clarification.

Reply:
I strongly recommend against returning the empty string for the controller property.... if makes sidetree look different than every other did method, and its not guaranteed to work accross JSON-LD implementations... the controller property should match the didDocument.id.

Reply:
Thanks for catching it. I will have a PR out fixing this soon.